indico/web/client/styles/modules/_registrationform.scss

```diff
@@ -918,6 +918,15 @@ textarea.regFormDescriptionInputMgmt {
     width: 15px;
     cursor: move;
   }
+
+  .disabled-row {
+    background-color: lightcoral;
```

Comment:
This color is a bit strong. Maybe try something from the `_palette.scss`?

This doesn't look bad IMHO

```scss
  .disabled-row {
    background-color: $light-red;

    input[name='caption'] {
      text-decoration: line-through;
    }
  }

  .disabled-row:hover {
    td {
      background-color: darken($light-red, 5%);
    }
  }
```